internal/client/client_test.go

```diff
@@ -19,7 +19,8 @@ func TestClientNewGitea(t *testing.T) {
 	ctx := &context.Context{
 		Config: config.Project{
 			GiteaURLs: config.GiteaURLs{
-				API: "https://git.dtluna.net/api/v1",
+				API:      "https://git.dtluna.net/api/v1",
+				Download: "https://git.dtluna.net",
```

Comment:
does this mean our tests actually hit this gitea server? 

if so, I would say we should change them, if not, I would say to set a fake address...

Reply:
@caarlos0 this was not added by me - tested and found out that `TestClientNewGitea` rely on it

should I change it to gitea.com ?

Reply:
I think its better yes, and maybe add a TODO to change this to a mocked http server...